nav2_simple_commander/README.md

```diff
@@ -102,8 +103,8 @@ The `nav2_simple_commander` has a few examples to highlight the API functions av
 
 - `example_nav_to_pose.py` - Demonstrates the navigate to pose capabilities of the navigator, as well as a number of auxiliary methods.
 - `example_nav_through_poses.py` - Demonstrates the navigate through poses capabilities of the navigator, as well as a number of auxiliary methods.
-- `example_waypoint_follower.py` - Demonstrates the waypoint following capabilities of the navigator, as well as a number of auxiliary methods required.
-
+- `example_waypoint_follower.py` - Demonstrates the waypoint following capabilities of the navigator, as well as a number of auxiliary methods.
+- `example_follow_path.py` - Demonstrates the path following capabilities of the navigator, as well as a number of auxiliary methods.
```

Comment:
Ditto on the nav2 website page

Reply:
Updated the required changes at this PR https://github.com/ros-planning/navigation.ros.org/pull/272